### ISSUE TITLE
Loading data in image form

### ISSUE THREAD

**Issue by Evizero** [johnmyleswhite/MNIST.jl#16]
Hey @johnmyleswhite !

Sorry for the noise, I know you are very busy.

I am currently working on a julia centered paper where I want to use MNIST for a tutorial purpose, which lead me here. I wanted to ask you if you are still interested and have the time to look over / merge PRs? Or, in the case you don't, that maybe I could convince you allow me to help out maintaining it (or donating to JuliaML)?

Basically, I would like to introduce additional functions to load the images in array / tensor form, to conveniently interface with Images.jl and Plots.jl (and the upcoming image views in 0.5), and also a LICENSE file. I don't think I would have to break anything, and I would probably also take the time to optimize the loading functions a bit.

Thoughts?

**Comment by johnmyleswhite** [comment]
I'd like to keep this package minimal, so I'd be happy to merge loading images into tensors, but would prefer that a separate package support integration with plotting tools. Is that ok?

I can add a LICENSE file, although I've always been cautious since I'm not totally sure I understand the licensing terms of the data being used by this package.

**Comment by Evizero** [comment]
> prefer that a separate package support integration with plotting tools. Is that ok?

Sure, sure. I don't mean introduce dependencies! I am purely talking about array structure convention to make conversion trivial. So the plan at least. regardless, I share your opinion and will do my best to follow that mantra

> I can add a LICENSE file

I would feel a lot better about that since I would like to use this package in a couple of places. Sadly, my understanding of licensing is worse than yours. Since this is a science dataset I would guess clearly pointing to the source in all sorts of places should put one into a safe spot?
